src/SpectatorView.Unity/Assets/SpectatorView/Materials/BlueButton.png.meta

```diff
@@ -119,7 +119,7 @@ TextureImporter:
     outline: []
     physicsShape: []
     bones: []
-    spriteID: 
+    spriteID: 9b3ef1ad36c71cb43a93ce922c44f2a6
```

Comment:
It looks like a lot of meta files have been updated here. Can they be reverted or is this changed needed?

Reply:
Those meta files get changed every time I open Unity, which I think means they are out of date. Somebody should fix the meta files. However, I reverted them to keep this PR focused.